.github/workflows/run-release-main.yml

```diff
@@ -13,9 +15,12 @@ jobs:
     strategy:
       matrix:
         python-version: [3.9]
+        release-version: [1.1.1]
```

Comment:
@cachukwu so is the workflow that we'll need to update the release version in both the _init_.py & in this github action? 

Could we use something like this to do this automatically? https://stackoverflow.com/questions/58177786/get-the-current-pushed-tag-in-github-actions

Maybe an improvement we can do in the future or a follow up PR.

Reply:
Yeah, I tried that (stackoverflow thread) yesterday. Didn't work as expected so I took this path. I want to come back to it in the future